runtime/src/error.c

```diff
@@ -45,19 +49,52 @@
 #include <libunwind.h>
 
 #ifdef __linux__
+#include <dlfcn.h> // for dladdr
 // We create a pipe with addr2line and try to get a line number
 // Currently the precise line number works only on linux64
```

Comment:
Is this comment still accurate, now that dladdr is being used?

Reply:
yes, we use dladdr to get the address to pass to addr2line. addr2line is not commonly available on non-linux.

Reply:
(we could look at getting this path working on Mac OS X if a user installed addr2line but it seems by then we probably should be doing something smarter anyway).

Reply:
gotcha, thanks